src/gluonnlp/model/bert.py

```diff
@@ -665,6 +702,61 @@ def bert_24_1024_16(dataset_name=None, vocab=None, pretrained=True, ctx=mx.cpu()
                           pretrained_allow_missing=pretrained_allow_missing, **kwargs)
 
 
+def ernie_12_768_12(dataset_name=None, vocab=None, pretrained=True, ctx=mx.cpu(),
+                    root=os.path.join(get_home_dir(), 'models'), use_pooler=True, use_decoder=True,
+                    use_classifier=True, pretrained_allow_missing=False, **kwargs):
```

Comment:
remove pretrained_allow_missing from the API

Reply:
Maybe we could just reuse `bert_12_768_12`? the dataset name 'baidu_ernie_uncased' should make it clear that it's the ERNIE model.

Reply:
ERNIE has max_length=513, activation=relu, layer_norm_eps=1e-5 which is a barrier